index.html

```diff
@@ -58,84 +56,84 @@ <h2>Interessen</h2>
       </section>
     </section>
  
-    <section id="biographie" class="standard-margin border-top">
-      <h2>Biographie</h2>
-          <p class="biographie-text">Im Jahre 1982 kam ich als echtes Nordlicht in Hamburg zur Welt. Beruflich zog es mich 
+    <section class="bio bio__border bio__margin">
```

Comment:
Du nutzt hier bio__boder & margin obwohl du die klasse "bio" nicht wieder benutzt und bio border&margin auch nicht. Du könntest bio border & margin einfach mit in das bio styling reinnehmen